### ISSUE TITLE
Unable to make swipe methods to make work with stack

### ISSUE THREAD

**Issue by glebcha** [chitezh/react-native-swiper-animated#10]
```
    onRightSwipe: () => {},
    onLeftSwipe: () => {},
    onRemoveCard: () => {},
```

**Comment by chitezh** [comment]
Hey, @glebcha Are you on stack mode or non-stack?

**Comment by glebcha** [comment]
@chitezh sorry, changed issue topic. I'm in stack mode. Currently tested and verify that methods work only for non-stack mode, but why?

**Comment by craigdub** [comment]
Running into same issue. Did you find a solution to this? @glebcha @chitezh

**Comment by glebcha** [comment]
@craigdub the only solution is to fork and send PR :trollface:

**Comment by craigdub** [comment]
lol I was considering... Unless you know of a similar lib that gets it done? @glebcha

**Comment by chitezh** [comment]
Thanks, @glebcha @craigdub. A PR would be appreciated 🥇

**Comment by chitezh** [comment]
Fixed by #14
